SonarRunner.Shim/PropertiesFileGenerator.cs

```diff
@@ -204,6 +207,33 @@ private static string TryGetCodeCoverageReport(ProjectInfo project, ILogger logg
             return vsCoverageReport;
         }
 
+        private static AnalysisProperties GetAnalysisProperties(AnalysisConfig config, ILogger logger)
```

Comment:
The name and signature of this method (GetAnalysisProperties) suggests is behaves like query. However it does some writing, so the name is slightly misleading.